### PR TITLE
Metadata: BigData: utilize granulate-utils' BigData metadata collection

### PR DESCRIPTION
**Pull request by roi-granulate**
## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the relevant documentation.
- [ ] I have added tests for new logic.

### CODE REVIEW

**Comment by Jongy** [comment]
@roi-granulate please rebase

File: gprofiler/metadata/metadata_collector.py [hunk 1]
```diff
@@ -16,7 +17,7 @@ def get_static_metadata(spawn_time: float, run_args: UserArgs) -> Metadata:
     formatted_spawn_time = datetime.datetime.utcfromtimestamp(spawn_time).replace(microsecond=0).isoformat()
     static_system_metadata = get_static_system_info()
     cloud_metadata = get_static_cloud_instance_metadata(logger)
-    bigdata = get_bigdata_info()
+    bigdata = run_in_ns(["mnt"], get_bigdata_info)
```
Comment (Jongy) [inline gprofiler/metadata/metadata_collector.py:20]: Btw - previously it wasn't running in host mnt NS, why changed now?

**Review by Jongy — CHANGES_REQUESTED** [review]
1 question

File: gprofiler/metadata/metadata_collector.py (hunk 1 above)
Comment (roi-granulate) [reply gprofiler/metadata/metadata_collector.py:20]: to support cases where we see spark deployed over K8s

Comment (Jongy) [reply gprofiler/metadata/metadata_collector.py:20]: To support spark over k8s

**Review by Jongy — APPROVED** [review]

**Pull request merged**